src/controllers/confirmation.controller.ts

```diff
@@ -0,0 +1,65 @@
+import { NextFunction, Request, Response } from "express";
+import {formatDateForDisplay} from "../client/date.formatter";
+import logger from "../logger";
+import {PTFCompanyProfile} from "../model/company.profile";
+import * as templatePaths from "../model/template.paths";
+import * as sessionService from "../services/session.service";
+import {ALREADY_SUBMITTED, COMPANY_NUMBER, COMPANY_PROFILE, SIGN_IN_INFO, USER_PROFILE} from "../session/keys";
+import {IUserProfile} from "../session/types";
+
+const ACCOUNTS_EXT_DEADLINE_IN_DAYS: number = 28;
```

Comment:
I still think that we should ask an analyst to update the story with the number of days. Without it, we have zero traceability to requirements.

Reply:
These are really just placeholders for the time being; they can still be changed.

Reply:
Ok but what I'm struggling with is that there's no requirements/story around any of this logic to extend the deadline, including the number of days applied - at least it's not in the story that this PR relates to. So, should this code even be in this PR if not relevant now/yet? I suppose you could clarify it by adding a TODO comment to the code, but then what JIRA ticket will this hang off?

Reply:
I can add a TODO that's the best idea. As for which jira ticket - this returns me to the point that there is only one ticket to cover the controller - I will reference case LFA-1169 which deals with the confirmation for company is still reqired as this is what it relates to.

Reply:
Ok, thanks. At least we then have some kind of reference and it's a reminder to check this later.

Reply:
I would tend to agree with Martin, in lfa-1365, or it's parent story, lfa-1166. There's no story for any logic in this controller barring retrieving company name from the session, probably best to not include any logic until there is a story for it, rather than placeholder logic we risk leaving in that's not founded in user requirements.
But we should decide this as a team

Reply:
Alright I'll remove this.